src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Heading,
+  Notification as DSNotification,
+} from "@nypl/design-system-react-components"
+
+/**
+ * Custom RC Notification component that internally uses the DS `Notification`
+ * component. The header level and aria-label are intentionally hard-coded.
+ * For now, this notification should only display on the home and search result
+ * pages and that is controlled in the `Layout` component.
+ */
+const Notification = ({ notification = "" }) => {
+  if (!notification) return null
+
+  return (
+    <DSNotification
+      aria-label="Research Catalog Alert"
+      notificationHeading={
+        <Heading level="h2">New Service Announcement</Heading>
```

Comment:
Made this an h2 but it still _displays_ as an h4. This was done to meet the accessibility heading hierarchy:

<img width="235" alt="Screen Shot 2024-02-08 at 5 31 14 PM" src="https://github.com/NYPL/research-catalog/assets/1280564/b5de6a3c-6d08-4cc2-953a-58f464887c6f">